meta-ostro/conf/distro/ostro.conf

```diff
@@ -22,6 +22,20 @@ SDK_UPDATE_URL = "${OSTRO_SDK_UPDATE_URL}"
 # The default setting can be overridded in local.conf
 OSTRO_SDK_UPDATE_URL ??= "https://download.ostroproject.org/builds/ostro-os/latest/sdk-data/${MACHINE}"
 
+# The information about where to find version information and actual
+# content is needed in several places:
+# - the swupd client in the image gets configured such that it uses that as default
+# - swupd server needs information about the previous build
+#
+# The version URL determines what the client picks as the version that it updates to.
+# The content URL must have all builds ever produced and is expected to also
+# have the corresponding version information.
+SWUPD_VERSION_URL ?= "https://download.ostroproject.org/updates/ostro-os/milestone/${MACHINE}/${SWUPD_IMAGE_PN}"
+SWUPD_CONTENT_URL ?= "https://download.ostroproject.org/updates/ostro-os/builds/${MACHINE}/${SWUPD_IMAGE_PN}"
```

Comment:
@kad what you describe is the current deployment model. In this model, doing incremental builds (= reusing results from a previous build) is impossible because it isn't guaranteed that the content of the previous build gets published. It happened to work only because support for incremental builds was missing, which (as a side effect) made each build self-contained.

This should be changed because it is inefficient. I know it has implications for the Ostro CI and release process, which needs to be discussed should we ever do another release...